api/src/main/java/org/openmrs/module/Module.java

```diff
@@ -644,7 +648,15 @@ public Document getConfig() {
 	public void setConfig(Document config) {
 		this.config = config;
 	}
-	
+
+	public String getConfigVersion() {
```

Comment:
Can we put `@since` annotations on these? https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#New-Public-Methods-and-Classes